src/include/com/amazonaws/kinesis/video/webrtcclient/Stats.h

```diff
@@ -0,0 +1,321 @@
+/*
+ * Profiler public include file
+ */
+#ifndef __KINESIS_VIDEO_WEBRTCCLIENT_STATS_INCLUDE__
+#define __KINESIS_VIDEO_WEBRTCCLIENT_STATS_INCLUDE__
+
+#pragma once
+
+#ifdef  __cplusplus
+extern "C" {
+#endif
+
+////////////////////////////////////////////////////
+// Public headers
+////////////////////////////////////////////////////
+
+#define MAX_CANDIDATE_ID_LENGTH                   8U
+#define MAX_ADDRESS_LENGTH                       16U
+#define MAX_RELAY_PROTOCOL_LENGTH                 8U
+#define MAX_TLS_VERSION_LENGTH                    8U
+#define MAX_DTLS_CIPHER_LENGTH                    64U
+#define MAX_SRTP_CIPHER_LENGTH                    64U
+#define MAX_TLS_GROUP_LENGHTH                     32U
+#define MAX_PROTOCOL_LENGTH                       8U
+
+#define MAX_STRING_LENGTH                         255U
+
+/**
+ * @brief Type of Stats object requested by the application
+ * Reference https://www.w3.org/TR/webrtc-stats/#rtcstatstype-str*
+ */
+typedef enum {
+    CANDIDATE_PAIR, //!< ICE candidate pair statistics related to the RTCIceTransport objects
```

Comment:
We normally prefix the enum values with the enum name for the same reason - name collisions. So for example this one will be RTS_STAT_TYPE_CANDIDATE_PAIR. Same for all of the entries

Reply:
Got it. The reason I named the enums this way is to have it consistent with the spec. Let me rename all the enums with the prefix.